ftrackhq/ftrack-connect-pipeline-definition
# feat: Backlog/unreal/project context/shot publisher

Resolves : 

* CLICKUP-865bkucw2


This PR has been tested on :



## Changes

- Give artist ability to select a master sequence with shots, and publish them with batch publisher as image sequences and reviewables.

## Test

_Note: Must be tested with corresponding branch on definition, qt and unreal._

1. Create a master sequence with proper animated shots in Unreal.
2. Render the shots to a folder, with {shot} suffix to path.
3. Select the sequence in Level and run the ftrack Batch publisher.
4. Select a sequence from ftrack, and the folder rendered to (capture folder).
5. Run the batch publisher, shots should be created in ftrack with image sequence/movie asset versions.
